README.md

```diff
@@ -67,6 +67,11 @@ jobs:
           # These are indicated by a dollar sign and curly braces (`${placeholder}`).
           # Please refer to this action's README for all available placeholders.
           # pull_title: "[Backport ${target_branch}] ${pull_title}"
+
+          # Optional
+          # Regex pattern to match github labels which will be copied from the
+          # original PR to the backport PR.
```

Comment:
Let's clarify what happens to the backport labels. Also please note that the documentation generally uses the term `pull request` over `PR`.
```suggestion
          # Regex pattern to match github labels which will be copied from the
          # original pull request to the backport pull request. Labels matching
          # `label_pattern` will not be copied.
```

Please also apply these changes to `action.yml`